doc/site/index.tex

```diff
@@ -541,67 +592,78 @@ \subsection{CPANM Installation}\label{get.cpanm}
+and when used in conjunction with Ghostscript.
+
+\paragraph*{ImageMagick Versions} for the ImageMagick binaries and the Perl binding must match;
+often those available from CPAN are out-of-sync.
+Ideally, you should install the current versions of both ImageMagic and the perl binding
+from your OS/platform distribution, if available.
+Next best solution is to get the same versions from the same source
+following the instructions at \href{https://www.imagemagick.org/}{ImageMagick}
+to download and install the latest version of ImageMagick being sure to enable
+and build the Perl binding along with it.
 
-%============================================================
-\section{Documentation}\label{docs}
-If you're lucky, all that should be needed to convert
-a \TeX\ file, \textit{mydoc}\texttt{.tex} to XML, and
-then to HTML would be:
+\paragraph*{Installing ImageMagick under Windows}\label{get.windows.imagemagick}
+seems to work best by downloading and installing the main ImageMagick binary
+appropriate for your Windows system
+from \href{https://imagemagick.org/script/binary-releases.php#windows}{ImageMagick}.
+Then fetch the \texttt{PerlMagick} tarball \emph{with the same version} from
+\href{https://imagemagick.com/download/perl/}{ImageMagick/perl}.
+Use the following commands to compile and install the PerlMagick,
+with X.XX being the version you downloaded:
 \begin{lstlisting}[style=shell]
-   latexml --dest=%mydoc%.xml %mydoc%
-   latexmlpost --dest=%somewhere/mydoc%.html %mydoc%.xml
+tar -zxvf PerlMagick-X.XX.tar.gz
+cd PerlMagick-X.XX
+perl Makefile.PL
+dmake
+dmake test
+dmake install
 \end{lstlisting}
-This will carry out a default transformation into HTML5,
-which represents mathematics using MathML.  Using an
-extension of xhtml would generate XHTML including MathML.
-Adding the option \verb|--format=html4| to the call to \verb|latexmlpost|
-will generate HTML (version 4), using images to represent the math.
 
-If you're not so lucky, or want to get fancy, well \ldots dig deeper:
-\begin{description}
-\item[\href{manual/}{LaTeXML Manual}] (\href{manual.pdf}{\PDFIcon}).
-\item[\href{manual/commands/latexml.html}{\texttt{latexml}}]
-    describes the \texttt{latexml} command.
-\item[\href{manual/commands/latexmlpost.html}{\texttt{latexmlpost} command}]
-   describes the \texttt{latexmlpost} command for postprocessing.
-\end{description}
-
-% Possibly, eventually, want to expose:
-%   https://www.mathweb.org/wiki/????
-% But, it doesn't have anything in it yet.
+\paragraph*{ImageMagick Security} is also an issue, due to the possibility of embadding
+malicious code in postscript, pdf and other image formats.
+For a certain range of versions of ImageMagic and Ghostscript, a security policy was set
+which inhibited processing of pdf files; the problem appears to be fixed recent versions.
+If you see \texttt{operation not allowed by the security policy}, see the discussion
+at \href{https://stackoverflow.com/a/53180170}{stackoverflow}.
```

Comment:
@dginev This is where I address the issue, and suggest using the most recent versions, if you can. I also point to the stack overflow discussion where you can figure out if you need to do something special and how to do it.  It is way out of scope here to get into explaining how to modify the policy files of random OS's, however.

Reply:
Ok, so the security problem in ghostscript is fixed in recent versions, but the actual usability problem remains - latexml is broken by default for image conversion in ubuntu/debian systems (at the least).

All I ask is that we keep the issue in question open until the default state of image conversion on those OSes is "working", however we manage to achieve that.